src/ui/components/NetworkMonitor/utils.ts

```diff
@@ -33,39 +34,55 @@ export type RequestSummary = {
   url: string;
 };
 
+// From https://github.com/RecordReplay/gecko-dev/blob/webreplay-release/devtools/server/actors/replay/network-helpers.jsm#L14
 export const REQUEST_TYPES = {
+  document: "HTML",
```

Comment:
Might be worth defining
```
enum CanonicalRequestType {
  html,
  // ...
};
```
so `REQUEST_TYPES` is a `Record<string, CanonicalRequestType>` lookup, and we can avoid relying on the human-readable string as the key in `REQUEST_ICONS`?

Reply:
Yep, good idea.